pkg/webhook/validation.go

```diff
@@ -133,8 +133,14 @@ func (v *validator) validateBundle(ctx context.Context, bundle *trustapi.Bundle)
 				unionCount++
 			}
 
+			if source.DefaultPackage {
+				unionCount++
+			}
+
```

Comment:
note to self: courtesy of @inteon - this will change to if `source.DefaultPackage != nil`. What if the user sets DefaultPackage to true in one source and false in another? Consider rejecting in that case.

Reply:
I think this isn't worth checking for. We already [check](https://github.com/cert-manager/trust-manager/blob/db71205da2310f2d389e18e74dcec9526c9ffa7f/pkg/webhook/validation.go#L106) if multiple fields are set.

If someone chose to explicitly write `defaultPackage: false` alongside a different source, I think we should treat that the same as we would if DefaultPackage were nil.